src/main/webapp/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <script src = "script.js"></script>
```

Comment:
Please change the name of the javacsipt file to a more suggestive one.
With will deal with lots of functionalities implemented with javacsript, it will be confusing to have such general filenames.

Reply:
I changed the name to `userVerification.js` since it verifies user's authentication and it the user already has data stored inside the database.